core/src/scene/styleContext.cpp

```diff
@@ -97,76 +97,64 @@ void pushYamlScalarAsJsObject(duk_context* ctx, const YAML::Node& node) {
     }
 }
 
-void StyleContext::parseSceneGlobals(const YAML::Node& node, const std::string& key, int seqIndex,
-                                     duk_idx_t dukObject) {
+void pushYamlScalarAsJsFunctionOrString(duk_context* ctx, const YAML::Node& node) {
+    auto scalar = node.Scalar().c_str();
+    duk_push_string(ctx, scalar); // Push function source.
+    duk_push_string(ctx, ""); // Push a "filename".
+    if (duk_pcompile(ctx, DUK_COMPILE_FUNCTION) != 0) { // Compile function.
+        auto error = duk_safe_to_string(ctx, -1);
+        LOGW("Compile failed in global function: %s\n%s\n---", error, scalar);
+        duk_pop(ctx); // Pop error.
+        duk_push_string(ctx, scalar); // Push property as a string.
+    }
+}
 
+void StyleContext::parseSceneGlobals(const YAML::Node& node) {
     switch(node.Type()) {
-        case YAML::NodeType::Scalar:
-            if (key.size() == 0) {
-                pushYamlScalarAsJsObject(m_ctx, node);
-                duk_put_prop_index(m_ctx, dukObject, seqIndex); // push
+    case YAML::NodeType::Scalar:
```

Comment:
We don't indent these consistently, in `styleParam.cpp` alone you can see both indented and un-indented case labels. However, our "official" style is to _not_ indent case labels (https://github.com/tangrams/tangram-es/blob/master/.clang-format#L27)